deploy/dev/5_load_environment.sh

```diff
@@ -23,7 +35,9 @@ function configure_secret() {
 
   echo "Create secret k8s-secret"
   $cli_with_timeout create -f $CONFIG_DIR/k8s-secret.yml
+}
 
+create_k8s_resources() {
   create_secret_access_role
 
   create_secret_access_role_binding
```

Comment:
these 2 method calls appear in tons of places. i think we can move the function `create_k8s_resources` to `utils.sh` and use it throughout the repos.

Also, i would change `create_k8s_resources` to something more specific like `define_secrets_resource_rbac`

Reply:
> these 2 method calls appear in tons of places. i think we can move the function create_k8s_resources to utils.sh and use it throughout the repos.

They are already in utils so I will just delete the wrapping function and just call them individually